docs/docs-content/integrations/ngrok.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_label: "ngrok"
+title: "ngrok"
+description: "Learn about using ngrok Kubernetes Ingress to access applications and services in Spectro Cloud."
+hide_table_of_contents: true
+type: "integration"
+category: ["ingress", "kubernetes"]
+sidebar_class_name: "hide-from-sidebar"
+logoUrl: ""
+tags: ["packs", "ngrok", "network", "kubernetes"]
+---
+
+The ngrok [Ingress Controller for Kubernetes](https://github.com/ngrok/kubernetes-ingress-controller) is the official
```

Comment:
@joelhans the word "official" raises a minor concern. Can we change this sentence so it's more apparent that it's ngrok's official ingress controller? Right now, it could potentially be perceived that this is our official ingress controller or the Kubernetes ecosystem's official controller.